src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,22 @@
+package teammates.test.driver;
+
+import teammates.common.util.StringHelper;
+
+/**
+ * Holds String-related helper functions used only in tests.
```

Comment:
On second thoughts, `Holds additional methods for {@link StringHelper} used only in tests.`